reactor-netty-http/src/main/java/reactor/netty/http/server/MicrometerHttpServerMetricsRecorder.java

```diff
@@ -37,9 +43,24 @@
 final class MicrometerHttpServerMetricsRecorder extends MicrometerHttpMetricsRecorder implements HttpServerMetricsRecorder {
 
 	final static MicrometerHttpServerMetricsRecorder INSTANCE = new MicrometerHttpServerMetricsRecorder();
+	private final static String HTTP = "http";
```

Comment:
maybe rename to `PROTOCOL_VALUE_HTTP` ? consider also adding a `PROTOCOL` tag name constant.

Reply:
oh I see... the URI tag is set to the protocol via the constructor and meter builders, for cases where the actual URI is unknown / non-applicable.